File: 2023/domaci/08-nasobilka/volfmatous/main.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
});

let lines = 0;
let n, p;

rl.on("line", (line) => {
    lines++;
    switch (lines) {
        case 1:
            n = parseInt(line);
            return;

        case 2:
            p = parseInt(line);
            break;
    }

    printMultiplicationTable(n, p);
});

function printMultiplicationTable(n, count) {
    for (let multiplier = 1; multiplier <= count; multiplier++) {
        console.log(n * multiplier);
    }
}

File: 2023/domaci/08-nasobilka/volfmatous/main.cs
using System;

internal class Program
{
    static void Main(string[] args)
    {
        int n = ReadInt();
        int p = ReadInt();

        PrintMultiplicationTable(n, p);
    }

    static void PrintMultiplicationTable(int n, int count)
    {
        for (int multiplier = 1; multiplier <= count; multiplier++)
        {
            Console.WriteLine(n * multiplier);
        }
    }

    static int ReadInt()
    {
        return int.Parse(Console.ReadLine());
    }
}
